indico/modules/rb_new/client/js/common/roomSearch/queryString.js

```diff
@@ -85,6 +85,21 @@ export const rules = {
             }
         }
     },
+    feat: {
+        validator: () => true,
+        stateField: {
+            serialize: ({filters: {features}}) => features,
+            parse: (value, state) => {
+                if (!Array.isArray(value)) {
+                    value = [value];
+                }
+                if (!state.filters) {
```

Comment:
When can it be `true`? Don't we set it to an empty object?

Reply:
just copied from equipments tbh.

Reply:
It's based on an [empty object](https://github.com/indico/redux-router-querystring/blob/master/src/validation.js#L19), which then gets merged into the redux state later.